docs/src/usage/distributed.rst

```diff
@@ -5,10 +5,18 @@ Distributed Communication
 
 .. currentmodule:: mlx.core.distributed
 
-MLX utilizes `MPI <https://en.wikipedia.org/wiki/Message_Passing_Interface>`_ to
```

Comment:
A thought on the usage guides.. I would probably group the two distributed usage guides into a single file (and maybe make `launching` another sub-heading on the right toc:

<img width="195" alt="Screenshot 2025-02-20 at 4 56 49 PM" src="https://github.com/user-attachments/assets/e55b5b2d-693f-4aa9-8aab-ca491084f841" />

Alternatively we could put them as two sub-items under a distributed top-level item which would let you keep them as separate files:

```
usage/
  distributed/
    distributed_communication/
    launching.../
```

But some kind of grouping in the usage guide would be good to keep it a bit organized.